owlbot.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import synthtool.languages.node as node
+
+node.owlbot_main(staging_excludes=
+    ['README.md', 'src/index.ts', 'linkinator.config.json', '.mocharc.json', 'package.json'],
```

Comment:
I can't think of a good reason for these to be excluded ....  I know they were like that in the original `synth.py`, but this doesn't seem right, or should at least have a good explanation.  I am ok with landing this for now, just please followup right after with a fix un-ignoring these.

Reply:
I un-ignored index.ts, but now the linter fails.  :-/